.circleci/config.yml

```
@@ -145,7 +145,9 @@ jobs:
       device:
         description: The device (e.g. "iPhone 12") to use when running tests.
         type: string
-        default: "iPhone 12"
+      scheme:
+        description: Xcode scheme with tests.
```

What do you think of this, to keep it consistent with the other parameter?

```suggestion
        description: Xcode scheme to use to run the tests.
```

---

Fully agree, thanks!